articles/introduction-to-editing.mdx

```diff
@@ -0,0 +1,138 @@
+
+<comp.Notice info_type="info">
+  If this is your first time creating a site on CloudCannon, or you need a refresher, check out our [Starter Guides](/documentation/guides/) — we have one for every SSG we support!
+</comp.Notice>
+
+In the Editing section of our documentation, we cover how to:
+
+* [Create a global configuration file](/documentation/articles/create-a-global-configuration-file/) — create the file at the heart of CloudCannon customization.
+* [Define your collections](/documentation/articles/define-your-collections/) — groups of related files that form the start of any new site.
+* Editing interfaces — the [Visual](/documentation/articles/introducing-the-visual-editor/), [Content](/documentation/articles/introducing-the-content-editor/), [Data](/documentation/articles/introducing-the-data-editor/), and [Source](/documentation/articles/introducing-the-source-editor/) Editors.
+* [Edit your files](/documentation/articles/edit-your-files/) — cooperate on file changes with live editing.
+* [Save your changes](/documentation/articles/save-your-changes/) — learn how to review changes made by your team and save back to your git repository.
+* [Configure your inputs](/documentation/articles/configure-your-inputs/) — configure your structured data to create an intuitive interface.
+* [Create a structure](/documentation/articles/create-a-structure/) — create predefined objects to add to your content.
+* Create a Snippet — create structured components in Markup using [Docusaurus](/documentation/articles/snippets-using-docusaurus-components/), [Eleventy](/documentation/articles/snippets-using-eleventy-shortcodes/), [Hugo Shortcodes](/documentation/articles/snippets-using-hugo-shortcodes/), [MDX](/documentation/articles/snippets-using-mdx-components/), or [Python Markdown](/documentation/articles/snippets-using-python-markdown/).
+
+Let’s briefly introduce a few of these topics.
+
+## The Global Configuration File
+
+Unlike a traditional CMS, you can configure CloudCannnon to suit your build tools and code. To maintain this flexibility, CloudCannon does not make assumptions about how you have configured your static site. CloudCannon will need you to clarify a few things about your SSG setup:
+
+* Where are your content files stored? How are they grouped?
+* Should CloudCannon allow the creation of new files? If so, how should they be created?
+* Do you want your team to be able to edit those files? If so, what options should they have?
+
+This setup information is stored in the CloudCannon global configuration file, saved in the root directory of your repository. Your configuration file can be one of several file types (i.e., `/cloudcannon.config.*`). When CloudCannon searches the root directory for your config file, it will use the first matching file from this list.
+
+1. `cloudcannon.config.json`
+2. `cloudcannon.config.yaml`
+3. `cloudcannon.config.yml`
+4. `cloudcannon.config.js`
+5. `cloudcannon.config.cjs`
```

Comment:
The `.js` and `.cjs` files are not supported in Jekyll. Not sure if that is worth a specific callout